package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "@gadget-client/app-with-no-user-model": "^1.5.0",
     "@gadget-client/bulk-actions-test": "^1.104.0",
-    "@gadget-client/related-products-example": "^1.853.0",
+    "@gadget-client/related-products-example": "^1.854.0",
```

Comment:
Was proving it was still an issue on the most recent version of the generated clients